[Untracked] Add Flutter docs warning against having multiple `BrazePlugin` instances (#8881)
braze-inc/braze-docs: _docs/_developer_guide/platforms/flutter/sdk_integration.md

Then, initialize an instance of the Braze plugin by calling `new BrazePlugin()` like in [our sample app](https://github.com/braze-inc/braze-flutter-sdk/blob/master/example/lib/main.dart).

{% alert important %}

> Be sure to allocate and use only one instance of the `BrazePlugin` in your Dart code to avoid undefined behaviors.

Reviewer: ```suggestion
To avoid undefined behaviors, only allocate and use a single instance of the `BrazePlugin` in your Dart code.
```
